### PR TITLE
Update re-send validation email view

### PR DESCRIPTION
**Pull request by vmariano**

### CODE REVIEW

File: lib/signin/view.js [hunk 1]
```diff
@@ -52,4 +52,30 @@ SigninForm.prototype.onsuccess = function() {
...
+  if (res.ok && JSON.parse(res.text).error === t("signin.error.email-not-valid")) {
+    page('/signup/resend-validation-email');
+  }
+
+  if (err || (res.body && res.body.error)) {
+    return this.errors([err || res.body.error]);
+  }
+
+  this.emit('success', res);
+};
```
Comment (gvilarino) [inline lib/signin/view.js]: @vmariano is this `response` function needed? I mean, it's implemented right in `FormView`, and with some error handling corrected. Maybe some input from @slifszyc would help?

Comment (gvilarino) [inline lib/signin/view.js]: This is the code I'm talking about: https://github.com/DemocracyOS/app/blob/development/lib/form-view/form-view.js#L133-L145

**Comment by vmariano** [comment]
>@vmariano is this response function needed? I mean, it's implemented right in FormView, and with some error handling corrected.

Yes, I was try to override from the original `response`. [1] Maybe there are a better workaround
Opinions?

[1] https://github.com/DemocracyOS/app/commit/72a4a636c705b56e42d6c08fbc9362ee9000f487#diff-d3e0c0a503cb474c9f9e229726439676R63

**Comment by gvilarino** [comment]
Yeah, the thing is that `response` is now obsolete after some refactor I did the other day. No worries; I'll adapt it. Also, I'll update the translation key [wich you seem to have forgotten](https://github.com/DemocracyOS/app/commit/72a4a636c705b56e42d6c08fbc9362ee9000f487#diff-193f73358ffc95a2f8935074d04b9da6R29)

**Comment by vmariano** [comment]
>Also, I'll update the translation key wich you seem to have forgotten

I think you are looking the code of the previus commit. Check (or update) to 
https://github.com/DemocracyOS/app/commit/3746252f58f4811ac4399ec5a059e4caf198fc87

**Comment by gvilarino** [comment]
Yea you're right, sorry about that.

File: lib/signin/view.js [hunk 2]
```diff
@@ -52,4 +52,30 @@ SigninForm.prototype.onsuccess = function() {
...
+ * @override from {FormView}
+ */
+SigninForm.prototype.response = function (err, res) {
+  if (!res.ok) {
+    return this.errors([JSON.parse(res.text).error]);
+  }
+
+  //Redirect if come from unverified email
+  //FIXME: this error detection mechanism is a little weird, we should avoid compare keys.
+  if (res.ok && JSON.parse(res.text).error === t("signin.error.email-not-valid")) {
```
Comment (gvilarino) [inline lib/signin/view.js]: About this; shouldn't we be using http response codes for handling these kind of weird situations?

**Pull request merged**